Documentation/ceph-monitoring.md

```diff
@@ -204,11 +204,11 @@ metadata:
   namespace: rook-ceph
 [...]
```

Comment:
Looks like this line is also not indented correctly.

Reply:
The `namespace:` line?    That's supposed to be in the `metadata:` block.

The continuation line `[...]`.  I can see it both ways, but it looks better indented, so I've indented it now.

Reply:
Thanks. Looks good now.